test/index.js

```diff
@@ -0,0 +1,176 @@
+const expect = require('expect');
+const {createRobot} = require('probot');
+const plugin = require('..');
+const commentEvent = require('./fixtures/issue_comment.created');
+
+describe('snooze', () => {
+  let robot;
+  let github;
+
+  beforeEach(() => {
+    robot = createRobot();
+
+    // Load the plugin
+    plugin(robot);
```

Comment:
Actually I think this promise is coming from probot itself!  However, that's a good point about visitors request for installations. I just completely `magic numbered` that for the time being, but mocking it would be really helpful.